cmake/util.cmake

```diff
@@ -156,3 +156,26 @@ function(add_python_test TEST_NAME)
         ${USE_VIRTUALENV_FOR_TEST} ${PYTHON_EXECUTABLE} ${ARGN}
         WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR})
 endfunction()
+
```

Comment:
Just mention there: gtest-parellel can reach the same goal. https://github.com/google/gtest-parallel